src/Responses/ProductCategoryResponse.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
```

Comment:
Make sure to remove the auto-generated IDE text since we know who touched what by using git.  In phpStorm, you can do this automatically in Settings -> Editor -> Code Style -> File and Code Templates > Includes (tab) and just completely wipe out the PHP File Header.